scripts/test/run-tests.sh

```diff
@@ -38,14 +38,18 @@ mkdir -p ${WORKSPACE}
 
 get_pytest_files
 
+prepare_ssh_keys
+
 if ! [ "$1" == "--all" -o "$1" == "--only" -a -n "$2" -o "$1" == "--prebuilt-image" -a -n "$3" ]; then
   usage
 fi
 
 test_result=0
 
 if [ "$1" == "--prebuilt-image" ]; then
-  run_tests "$2" "$3" || test_result=$?
+  run_tests "$2" "$3" \
+            "-k" "'not test_update'" \
+            || test_result=$?
```

Comment:
minor: do we really need to split this line? it is less than 70 characters long.

Reply:
Yeah, but I did it to be consistent with all the others below (the ones calling `convert_and_test`).